tests/libtest/lib650.c

```diff
@@ -62,6 +62,7 @@ int test(char *URL)
   struct curl_forms formarray[3];
   size_t formlength = 0;
   char flbuf[32];
+  long contentlength = 0;
```

Comment:
shouldn't this be a `curl_off_t` ?

Reply:
The deprecated `CURLFORM_CONTENTSLENGTH` used here uses `long`. I used the same cast sequence as in `formdata.c`. The newer `CURLFORM_CONTENTLEN` uses `curl_off_t`. If you want, I can switch to that in a separate commit before this one.